release/package/amd-builders-v4.yml

```diff
@@ -0,0 +1,64 @@
+version: "3"
```

Comment:
Here I would leave only operating system for which we only have to toolchain-v4, only if this is some backward compatibility think (not kill the existing builder containers until we fully migrate)